flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBValueState.java

```diff
@@ -64,7 +64,7 @@
 	 * @param stateDesc The state identifier for the state. This contains name
 	 *                           and can create a default state value.
 	 */
-	RocksDBValueState(ColumnFamilyHandle columnFamily,
+	public RocksDBValueState(ColumnFamilyHandle columnFamily,
```

Comment:
Ah I see, you're right. Could you make all of the State constructors public then? Just for consistency... 😃